commit 09db965a0bba3a4da017e5e6567dea6b886f60d0
Author: bertin <macportsuser@macports.org>
Date:   Thu May 22 23:41:30 2025 +0200

    /opt/local/site-ports/sysutils/MacPorts/files/patch-port-force-work-command.diff

diff --git a/src/port/port.tcl b/src/port/port.tcl
index 2ecf954db..0483e6ae4 100755
--- a/src/port/port.tcl
+++ b/src/port/port.tcl
@@ -3823,7 +3823,7 @@ proc action_portcmds { action portlist opts } {
                 work {
                     # output the path to the port's work directory
                     set workpath [macports::getportworkpath_from_portdir $portdir $portname]
-                    if {[file exists $workpath]} {
+                    if {[file exists $workpath] || [macports::global_option_isset ports_force]} {
                         puts $workpath
                     }
                 }
